gitlab/gitlab_exporter.py

```diff
@@ -95,6 +95,7 @@ def create_entity(blueprint: str, body: json, access_token: str):
     return response
 
 def get_all_merge_requests_from_gitlab():
+    # Gets all merge requests for this group and its subgroups.
     merge_requests_url = f"{GITLAB_URL}/merge_requests"
 
     response = requests.get(
```

Comment:
Is there a chance this request also requires parameters to handle pagination

Reply:
In this request - you can use the pagination params in case you want to restrict the list of merge requests
which is not our case